www/js/controller/articles.js

```diff
@@ -202,17 +202,12 @@ articles.controller('articles_controller',function($scope, $ionicModal, req, con
         //Don't really need a separate page since all we are just displaying the url
         //page for the article
         var options = {
-            location: 'no',
+            location: 'yes',
```

Comment:
I don't believe it is needed when using the system browser, but I didn't try passing them in at all. There also seems to be some correlation between using options and the system browser not open which I haven't quite figured out yet